src/Libraries/NavigationExperimental/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @see  https://github.com/facebook/react-native/blob/master/Libraries/NavigationExperimental/NavigationExperimental.js
+ */
+import createMockComponent from './../../components/createMockComponent';
```

Comment:
you don't need the leading `./` on this path, you can start straight in with `../`